scripts/perfbench/report_cmd.py

```diff
@@ -0,0 +1,137 @@
+import json
+import logging
+import sys
+from dataclasses import dataclass, field
+from pathlib import Path
+from statistics import median, stdev
+from typing import Dict, Iterable, List, Optional, TextIO, Tuple
+
+import click
+from perfbench_utils import (
+    RawReportEntry,
+    find_latest_prod_version,
+    get_raw_report_path,
+    work_dir_option,
+)
+
+
+# Do not show warn or improvement symbol if the delta is less than this percent
+MIN_DELTA_PERCENT = 1
```

Comment:
Suggestion : should this be an option of the cmd instead ? So we can fine tune it in the github workflow directly ?

Reply:
For info: I just changed it to a duration-based delta instead of a percent. Percentages were not useful for the durations we record. It would make more sense with longer operations I think.

I can turn this into an option indeed.